lmcache/storage_backend/local_backend.py

```diff
@@ -32,6 +38,23 @@ def __init__(
         self.chunk_size = config.chunk_size 
         self.config = config
         self.dict = {}
+        self.device = config.local_device
+        
+        self.put_queue = queue.Queue()
+        self.put_thread = threading.Thread(
+                target=self.put_worker, args=()
+            ) 
+        self.put_thread.start()
+        self.update_lock = threading.Lock()
+        
+        # FIXME(Jiayi): `use_pin_memory` and `dst_device` should be configged dynamically
```

Comment:
Maybe we should discuss this. There are a few alternatives for dst_device:
- Pass `dst_device` through vllm --> lmcache_vllm_driver --> lmcache engine --> lmcache backend
- Just assume dst_device is cuda, because a lot of lmcache functionalities depends on GPU, such as CacheGen.